_posts/2016-11-15-composer-mailchimp-subscriptions-v3-api.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Using MailChimp API v3.0 to subscribe users to email lists in Drupal custom modules"
+date: 2016-11-15
+author: Dan Murphy
+tags: drupal drupal-planet composer module-development mailchimp
+summary: How to use Composer along with the ThinkShout PHP library for MailChimp API v3.0 to easily subscribe users to mailing lists without the MailChimp contributed module.
+featured_image: "/blog/mailchimp_3_drupal_composer_3.jpg"
+featured_image_alt: "sheet music"
+featured_image_height: "983px"
+featured_image_width: "1474px"
+drupal_planet_summary: |
+  A tutorial that shows you how to use Composer and ThinkShout's new PHP library for MailChimp API v3.0 to easily subscribe users to mailing lists in Drupal custom modules without using the MailChimp contributed module. This is a follow-up to a previous post that used the old API, and also includes some new Drupal 8 specifics.
+---
+
+Earlier this year I wrote [a blog post](/2016/01/22/composer-mailchimp-subscriptions.html) showing how to use [Composer Manager](https://www.drupal.org/project/composer_manager) along with the Mailchimp API v2.0 PHP package to subscribe users to mailing lists in a Drupal 6 or 7 custom module without the need for the Mailchimp contributed module.
```

Comment:
Please write a few introductory sentences about how we use mailchimp for our clients

Something to the effect of "Our clients are often looking to reach their audiences via email campaigns, and one of the solutions we often recommend to manage this and provide good analytics on their users behavior is Mailchimp"

Once you've done that, please deploy it; it's ready.